src/SqlPersistence/ManifestOutput.cs

```diff
@@ -0,0 +1,153 @@
+                new VarcharSchemaProperty { Name = "Metadata", Length = "max", Mandatory = true },
+                new VarcharSchemaProperty { Name = "Data", Length = "max", Mandatory = true },
+                new VarcharSchemaProperty { Name = "PersistenceVersion", Length = "23", Mandatory = true },
+                new VarcharSchemaProperty { Name = "SagaTypeVersion", Length = "23", Mandatory = true },
+                new SchemaProperty { Name = "Concurrency", Type = "integer", Mandatory = true }
+            ];
+        }
+
+        public record SubscriptionManifest
+        {
+            public string TableName { get; init; }
+            //NOTE this is hardcoded so if the subscription script (Create_MsSqlServer.sq in ScriptBuilder/Subscriptions) changes this needs to be updated
+            public string PrimaryKey => "Subscriber, MessageType";
+            public IndexProperty[] Indexes => [];
+            public object[] TableColumns => [
+                new VarcharSchemaProperty { Name = "Subscriber", Length = "200", Mandatory = true },
+                new VarcharSchemaProperty { Name = "Endpoint", Length = "200", Mandatory = true },
+                new VarcharSchemaProperty { Name = "MessageType", Length = "200", Mandatory = true },
+                new VarcharSchemaProperty { Name = "PersistenceVersion", Length = "23", Mandatory = true }
+            ];
+        }
+
+        public record SchemaProperty
+        {
+            public string Name { get; init; }
+            public virtual string Type { get; set; }
+            public bool Mandatory { get; init; }
+        }
+
+        public record VarcharSchemaProperty : SchemaProperty
+        {
+            public override string Type => "string";
+            public string Length { get; init; }
+        }
+
+        public record BitSchemaProperty : SchemaProperty
+        {
+            public override string Type => "boolean";
+            public bool Default { get; init; }
+        }
+
+        public record IndexProperty
+        {
+            public string Name { get; init; }
+            public string Columns { get; init; }
+        }
+    }
+
+    protected override void Setup(FeatureConfigurationContext context)
+    {
+        var settings = context.Settings;
+        var endpointName = settings.Get<string>("NServiceBus.Routing.EndpointName");
+        //dialect information
+        if (!settings.HasSetting("SqlPersistence.SqlDialect"))
+        {
+            return;
+        }
+        var dialect = settings.GetSqlDialect();
+
+        var manifest = new PersistenceManifest
+        {
+            Dialect = dialect.Name,
+            //this applies to all queues/tables
+            Prefix = endpointName
+        };
+
+        //outbox information
+        if (settings.IsFeatureActive(typeof(SqlOutboxFeature)))
+        {
+            manifest.Outbox = new PersistenceManifest.OutboxManifest { TableName = dialect.GetOutboxTableName($"{endpointName}_") };
```

Comment:
should we be using the Prefix value here, so it's easier to contain any changes?